README.md

```diff
@@ -1,51 +1,98 @@
 # Kik Bot API #
-Use this library to develop bots for [Kik Messenger](https://www.kik.com) that are essentially automated humans.
+The Kik Bot API is a Python library developed to automate interactions on [Kik Messenger](https://www.kik.com) that are essentially automated humans.
 
-It basically lets you do the same things as the official Kik app by pretending to be a real smartphone client; It communicates with Kik's servers at `talk1110an.kik.com:5223` over a modified version of the [XMPP](https://xmpp.org/about/technology-overview.html) protocol.
+It's essentially a way to create bots that behave like humans on the platform. This library enables your bot to interact with the official Kik app by emulating a real smartphone client. It communicates with Kik's servers at `talk1110an.kik.com:5223` over a modified version of the [XMPP](https://xmpp.org/about/technology-overview.html) protocol.
+
+This library is ideal for developers, hobbyists, and businesses who want to build automated bots to interact with users, groups, and other bots on Kik.
+
+We do not endorse the use of this library for spamming or other malicious purposes. Please use this library responsibly.
+
+## Features ##
+With the Kik Bot API, you can:
+
+- Log in with kik username and password, retrieve user information (such as email, name, etc).
+- Fetch chat partners information
+- Send text messages to users/groups and listen for incoming messages
+- Send and receive 'is-typing' status
+- Send and receive read receipts
+- Fetch group information (name, participants, etc.)
+- Fetch past message history
+- Administer groups (add, remove or ban members, etc)
+- Search for groups and join them (experimental feature)
+- Receive media content: camera, gallery, stickers
+- Add a kik user as a friend
+- Send images (including GIFs, using a [Tenor](https://tenor.com/gifapi) API key)
+
+Sending videos or recordings is not supported yet.
 
-This is the new branch of this project and is recommended.
 ## Installation and dependencies ##
-First, make sure you are using **Python 3.6+**, not python 2.7. Second, just install it directly from GitHub:
+Make sure you have Python 3.8 or above installed on your system. You can install this library directly from GitHub:
 ```
 git clone -b new https://github.com/tomer8007/kik-bot-api-unofficial
 pip3 install ./kik-bot-api-unofficial
 ```
-## Usage ##
-Examples are a great way to understand things. A good place to start is the `examples/` directory. 
+## Quick Start Guide ##
+Here's a simple example of how to use the Kik Bot API:
```

Comment:
I think the example here is now a bit too long in lines, I like to keep it simple.

Reply:
I made it a bit simpler by removing on_group_message_received and on_login_error, but I kept them in simple_echo_bot.py  since most people use the bots for groups and it wont work without the captcha